core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -23,14 +23,21 @@
  * {@link RuntimeException} if the key is not found in both configurations..
  */
 public final class UnderFileSystemConfiguration {
+  private final boolean mReadOnly;
+  private final boolean mShared;
```

Comment:
Should these two also be part of `mUfsConf`? By adding these here, it makes this class look more like `MountConfiguration`.